test/Data.UnitTests/Camera/CameraFactoryTest.cs

```diff
@@ -0,0 +1,60 @@
+namespace Data.UnitTests.Camera
+{
+    using Business.Camera.Common.Models;
+    using Business.Camera.Register.Models;
+    using Data.Camera;
+    using NUnit.Framework;
+    using Shouldly;
+
+    public class CameraFactoryTest
+    {
+        private MCamera _model;
+        private ECamera _entity;
+        private MRegisterCamera _modelCreate;
```

Comment:
Would the name `_input` be more relevant here?

Reply:
Done